mtags/src/main/scala-2/scala/meta/internal/pc/PcSyntheticDecorationsProvider.scala

```diff
@@ -191,6 +192,15 @@ final class PcSyntheticDecorationsProvider(
 
     private def isNotInUnapply(vd: ValDef) =
       !vd.rhs.pos.isRange || vd.rhs.pos.start > vd.namePosition.end
+
+    /* If is left part of val definition bind:
+     * val <<t>> @ ... =
+     */
+    private def isValDefBind(vd: ValDef) = {
+      val (_, afterDef) = vd.pos.source.content.splitAt(vd.namePosition.end)
```

Comment:
Maybe we can move it to `PcSyntheticDecorationsProvider` so we can use `text` instead of `pos.source.content`?

Reply:
I don't want to move it for Scala 3, I'd have to pattern match to see it's a ValDef, and it's a bit inconsistent. I added a given for Scala 3 instead (with an opaque type). What do you think? Too much hassle?

Reply:
I think it's fine